src/App.tsx

```diff
@@ -3,10 +3,18 @@ import { BrowserRouter, Routes, Route } from "react-router-dom";
 import { NavigationBar } from "./Components/Navbar";
 import { HomePage } from "./Pages/Home";
 import { DonationPage } from "./Pages/Donation";
-import { FestivalPage } from "./Pages/Festival";
 import { Contact } from "./Pages/Contact";
-import { VisitUs } from "./Pages/VisitUs";
+import { VisitUs } from "./Pages/Visitus";
```

Comment:
```suggestion
import { VisitUs } from "./Pages/VisitUs";
```
Change file name if required

Reply:
resolved